### PR TITLE
[FRONTEND] init

### PR DESCRIPTION
**Pull request by mahanbeom**
review test

### CODE REVIEW

File: frontend/.browserslistrc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+
+# For the full list of supported browsers by the Angular framework, please see:
+# https://angular.io/guide/browser-support
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
+
+last 1 Chrome version
```
Comment (hayongwoon) [inline frontend/.browserslistrc:11]: test

File: frontend/.browserslistrc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+
+# For the full list of supported browsers by the Angular framework, please see:
+# https://angular.io/guide/browser-support
+
+# You can see what browsers were selected by your queries by running:
+#   npx browserslist
```
Comment (mahanbeom) [inline frontend/.browserslistrc:9]: test

**Review by mahanbeom — COMMENTED** [review]
etest

**Review by hayongwoon — COMMENTED** [review]
wht?

**Pull request merged**